### ISSUE TITLE
Remaining shields of Italy

### ISSUE THREAD

**Issue by claysmalley** [ZeLonewolf/openstreetmap-americana#498]
<img height="16" alt="Strada_Statale_16_Italia" src="https://user-images.githubusercontent.com/1732117/179310114-d7aa5d1e-3fc8-444f-a845-cad96f09b48b.svg" /> State, <img height="16" alt="Strada_Regionale_203_Italia" src="https://user-images.githubusercontent.com/1732117/179310116-28fc0a03-177d-4404-aeb4-b07338ee6d59.svg" /> regional and <img height="16" alt="Strada_Provinciale_47_Italia" src="https://user-images.githubusercontent.com/1732117/179310115-5c16cfca-a642-4003-b8df-e73c285dd8c0.svg" /> provincial highways all have white-on-blue rectangular shields. <img height="16" alt="Italian_traffic_signs_-_strada_comunale_2" src="https://user-images.githubusercontent.com/1732117/179310117-363c99a0-9da5-4148-a94b-c5d82a756db8.svg" /> Communal highways have black-on-white rectangular shields. 

<img height="16" alt="Nuova_Strada_Statale_2_Italia" src="https://user-images.githubusercontent.com/1732117/179310124-e21cc473-19b2-4ce6-a042-9a204e8e09f9.svg" /> NSA seems to be a provisional designation for newly constructed highways that have yet to be designated as a state, regional or provincial highway. 

<img height="16" alt="Traforo_T4_Italia" src="https://user-images.githubusercontent.com/1732117/179310123-b4d65e4a-07c0-4038-9a4f-040733d8839a.svg" /> Tunnel routes are currently tagged as motorways, though they would probably be better as a network of their own. Likewise, <img height="16" alt="Italian_traffic_signs_-_raccordo_autostradale_3" src="https://user-images.githubusercontent.com/1732117/179310120-f4b225d1-c82e-4e28-af74-54482b57b051.svg" /> motorway spurs are tagged under the same network as motorways, despite having a distinct white-on-green rectangular shield.

`ref` values in Italy can be [very long,](https://www.openstreetmap.org/relation/6769829) and we will need to change the maximum `ref` length.

----

Le strade statali, <img height="16" alt="Strada_Statale_16_Italia" src="https://user-images.githubusercontent.com/1732117/179310114-d7aa5d1e-3fc8-444f-a845-cad96f09b48b.svg" /> regionali <img height="16" alt="Strada_Regionale_203_Italia" src="https://user-images.githubusercontent.com/1732117/179310116-28fc0a03-177d-4404-aeb4-b07338ee6d59.svg" /> e provinciali <img height="16" alt="Strada_Provinciale_47_Italia" src="https://user-images.githubusercontent.com/1732117/179310115-5c16cfca-a642-4003-b8df-e73c285dd8c0.svg" /> hanno tutte simboli rettangolari bianchi su blu. Le strade comunali <img height="16" alt="Italian_traffic_signs_-_strada_comunale_2" src="https://user-images.githubusercontent.com/1732117/179310117-363c99a0-9da5-4148-a94b-c5d82a756db8.svg" /> hanno simboli rettangolari neri su bianco.

NSA <img height="16" alt="Nuova_Strada_Statale_2_Italia" src="https://user-images.githubusercontent.com/1732117/179310124-e21cc473-19b2-4ce6-a042-9a204e8e09f9.svg" /> sembra essere una designazione provvisoria per strade di nuova costruzione che devono ancora essere designate come autostrada statale, regionale o provinciale.

I percorsi dei trafori <img height="16" alt="Traforo_T4_Italia" src="https://user-images.githubusercontent.com/1732117/179310123-b4d65e4a-07c0-4038-9a4f-040733d8839a.svg" /> sono attualmente etichettati come autostrade, anche se probabilmente sarebbero migliori come una propria rete. Allo stesso modo, gli raccordi autostradali <img height="16" alt="Italian_traffic_signs_-_raccordo_autostradale_3" src="https://user-images.githubusercontent.com/1732117/179310120-f4b225d1-c82e-4e28-af74-54482b57b051.svg" /> sono contrassegnati nella stessa rete delle autostrade, nonostante abbiano distinti simboli rettangolari bianco su verde.

I valori `ref` in Italia possono essere [molto lunghi](https://www.openstreetmap.org/relation/6769829) e dovremo modificare la lunghezza massima di `ref`.

**Comment by ZeLonewolf** [comment]
In addition, there are alternate routes for the state roads which have suffixes:
bis = 2nd
ter = 3rd
qtr = 4th

The longest suffixes are 4 characters long (list: https://sites.google.com/site/roadnumberingsystems/home/countries/italy)

So, SS7qtr (a real road that I've driven on) is the 4th spur of SS7.  Conceivably there could be roads with a longer ref and suffix.

**Comment by 1ec5** [comment]
> Conceivably there could be roads with a longer ref and suffix.

Yes, the example in the original post is not unusual. [This way](https://www.openstreetmap.org/way/175701053) near Milan is part of [SPexSS412](https://www.openstreetmap.org/relation/16061339).
